naming-conventions.md

```diff
@@ -516,7 +509,7 @@ Enums are used to define a set of named constants. The following conventions sho
   
   - **Enum Definition**:
     ```typescript
-    export enum UserStatus {
+    export enum UserStatusEnum {
```

Comment:
Types in de name vind ik enkel acceptabel in dynamically typed languages zonder type hinting